app/uk/gov/hmrc/entrydeclarationintervention/controllers/EisInboundAuthorisedController.scala

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.entrydeclarationintervention.controllers
+
+import play.api.mvc._
+import uk.gov.hmrc.entrydeclarationintervention.config.AppConfig
+import uk.gov.hmrc.http.logging.Authorization
+import uk.gov.hmrc.play.bootstrap.controller.BackendController
+
+import scala.concurrent.{ExecutionContext, Future}
+
+abstract class EisInboundAuthorisedController(cc: ControllerComponents, appConfig: AppConfig)
+    extends BackendController(cc) {
+  def basicAuthorisedAction: ActionBuilder[Request, AnyContent] =
```

Comment:
Maybe rename to simply `authorisedAction` (as not really using Basic auth if the scheme named in the header is Bearer).